.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug report
+about: Submit a bug request
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+[//]: #  (Please do not include your AWS account or credentials as part of the bug request.)
+
+**Describe the Bug**
+
+[//]: #  (Please describe the behavior that you are seeing with as many details as possible including logs from AGC, CloudWatch snippets and CloudFormation errors if relevant.)
+
+
+**Steps to Reproduce**
+
+[//]: #  (Include a list of steps that can be used to reproduce the bug.)
+
+
+**Relevant Logs**
+
+[//]: #  (The logs that where obtained by running the commands here. Please run the command with "-v" so that we can see the verbose logs.)
+
+
+**Expected Behavior**
+
+[//]: #  (A clear and concise description of what you expected to happen.)
+
+
+**Actual Behavior**
+
+[//]: #  (A clear and concise description of what actually happened.)
+
+
+**Screenshots**
+
+[//]: #  (If applicable, add screenshots to help explain your problem.)
+
+
+**Additional Context**
+
+[//]: #  (Add any other context about the problem here.)
+
+
```

Comment:
It would be important to also know:

* Amazon Genomics CLI version
* Operating system and version
* Account activated with custom VPC
* Account activated with custom S3 Bucket